CMakeLists.txt

```diff
@@ -423,19 +423,21 @@ add_subdirectory("docs/doxygen")
 #
 # Note: does not check `tests` directory which depends too much on conditional
 # compilation
-file(GLOB_RECURSE ALL_SOURCE_FILES RELATIVE ${CMAKE_SOURCE_DIR}
-    src/*.cc src/*.c src/*.h)
-foreach(FILE ${ALL_SOURCE_FILES})
-    string (FIND ${FILE} "/isotp_conn/" EXCLUDE_DIR_FOUND)
-    if (NOT ${EXCLUDE_DIR_FOUND} EQUAL -1)
-      continue()
-    endif()
+if(BUILD_ISOTP)
```

Comment:
This is not really satisfactory, this check should run even if `BUILD_ISOTP` is off.

The solution is to put the `aktualizr_source_file_checks` outside the check, so that it's run unconditionally.

Reply:
Yes, I think you are right. I will modify it.